_people/enrico_siragusa.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Enrico Siragusa
+category: Contributor
```

Comment:
```suggestion
category: Alumni
```
?

Reply:
On the old page he was marked as contributor. I wasn't sure where to put him :)